theme-release-100rc1.rst

```diff
@@ -0,0 +1,115 @@
+.. post:: August 23, 2021
+    :tags: theme, release
+    :author: Anthony
+
+.. meta::
+    :description lang=en:
+        Information on the sphinx_rtd_theme pre-release, version 1.0.0rc1, and
+        following releases.
+
+Theme release 1.0.0rc1
+======================
+
+We are closing in on the next official release of `sphinx_rtd_theme`_, and
+we're happy to announce that release candidate version 1.0.0rc1 was
+`released to PyPI`_ last week! Project maintainers can contribute to the final
+release by `testing the release candidate <Testing_>`_ and raising any potential
+bugs on `our issue tracker <issue-tracker_>`_.
+
+The version of our next release was incremented to 1.0.0 due to several
+backwards incompatible changes, however this release also adds support for
+`Sphinx`_ 4, and fixes several issues stemming from `docutils`_ 0.17.
```

Comment:
I would invert this, to put the positive stuff at the beginning:

```suggestion
Version 1.0.0 adds support for `Sphinx`_ 4, fixes several issues stemming
from `docutils`_ 0.17, and contains several backwards-incompatible changes.
```

Reply:
Great point!